gatling-http/src/main/scala/io/gatling/http/request/builder/HttpRequestBuilder.scala

```diff
@@ -128,9 +128,9 @@ final case class HttpRequestBuilder(commonAttributes: CommonAttributes, httpAttr
   def multivaluedFormParam(key: Expression[String], values: Expression[Seq[Any]]): HttpRequestBuilder = formParam(MultivaluedParam(key, values))
 
   def formParamSeq(seq: Seq[(String, Any)]): HttpRequestBuilder = formParamSeq(seq2SeqExpression(seq))
-  def formParamSeq(seq: Expression[Seq[(String, Any)]]): HttpRequestBuilder = formParam(ParamSeq(seq))
+  def formParamSeq(seq: Expression[Seq[(Any, Any)]]): HttpRequestBuilder = formParam(ParamSeq(seq))
```

Comment:
What's the use case for allowing to pass something else that a String as the key?
We mostly want to be able to pass `formParamSeq(Seq("${foo}" -> 1))` (an EL String key) while still being able to pass `formParamSeq(Seq("bar" -> 2))` (a regular String key).

Reply:
Thanks) I understood my mistakes!